src/components/modules/toolbar/conversion.ts

```diff
@@ -47,9 +48,9 @@ export default class ConversionToolbar extends Module<ConversionToolbarNodes> {
   public opened = false;
 
   /**
-   * Available tools
+   * Available tools data
    */
-  private tools: { [key: string]: HTMLElement } = {};
+  private tools: {name: string; toolboxItem: ToolboxConfig; button: HTMLElement}[] = []
```

Comment:
`toolboxItem = toolboxConfig` — this looks not so clear because the Toolbox's config may consist of several items.